aiohttp/web_reqrep.py

```diff
@@ -240,6 +240,22 @@ def if_modified_since(self, _IF_MODIFIED_SINCE=hdrs.IF_MODIFIED_SINCE):
         return None
 
     @reify
+    def authorization(self, _AUTHORIZATION=hdrs.AUTHORIZATION):
+        """Parse the Authorization header and return the username and password,
+        or ``None`` if the header is absent or cannot be parsed.
+
+        :rtype:  :class:`aiohttp.helpers.BasicAuth`
+
+        """
+        header = self.headers.get(_AUTHORIZATION)
+        if header is not None:
+            try:
+                return BasicAuth.decode(header)
```

Comment:
Understood.

We could make `authorization` a method instead accepting a `encoding` parameter?